src/object/rigid_body_collision_groups.rs

```diff
@@ -0,0 +1,122 @@
+use ncollide::world::CollisionGroups;
+
+// internal reserved static group id
+const STATIC_GROUP_ID: usize = 29;
+
+/// TODO: better explanations or redirect user to `ncollide` documentation ?!?
```

Comment:
For the moment I suggest you just remove the TODO line and keep your rest of your comment. I will have to reword it when sensors will be added anyway.